ui/components/app/transaction-breakdown/transaction-breakdown.component.js

```diff
@@ -142,16 +149,19 @@ export default class TransactionBreakdown extends PureComponent {
         <TransactionBreakdownRow
           title={t('transactionHistoryEffectiveGasPrice')}
         >
-          <UserPreferencedCurrencyDisplay
-            className="transaction-breakdown__value transaction-breakdown__value--effective-gas-price"
-            type={PRIMARY}
-            value={effectiveGasPrice}
+          <CurrencyDisplay
+            className="transaction-breakdown__value"
+            data-testid="transaction-breakdown__effective-gas-price"
+            currency={nativeCurrency}
+            denomination={ETH}
+            numberOfDecimals={6}
+            value={hexGasTotal}
           />
```

Comment:
Did we decide that we don't want to use user preferred currency for transaction breakdowns?

Reply:
Oh, maybe we can. I thought it might not work, but I see now that `UserPreferencedCurrencyDisplay` uses `CurrencyDisplay` internally. I will update this

Reply:
Switched to using `userPreferencedCurrencyDisplay` in this commit f46f6dc37

I am not sure of the decisions behind the other usages of `Currency` display in this component, but we can address consistency later